src/app/student/dashboard/page.tsx

```
@@ -0,0 +1,58 @@
+            {showProfile ? 'Hide Profile' : 'Profile'}
+          </button>
+        </div>
+
+        {showProfile ? (
+          <UserProfile />
+        ) : (
+          <div className="bg-white rounded-lg shadow-md p-6">
+            <h2 className="text-xl font-semibold mb-4">Welcome to your Dashboard</h2>
+            <p className="text-gray-600">
+              Here you can view your courses, assignments, and other important information.
+              Click the "View Profile" button to see and edit your profile details.
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)